SPECS/rpm/rpm.spec

```diff
@@ -158,7 +147,6 @@ install -vm755 %{SOURCE1} %{buildroot}%{_libdir}/rpm/
 install -vm755 %{SOURCE2} %{buildroot}%{_libdir}/rpm/
 
 pushd python
-python2 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
 python3 setup.py install --skip-build --prefix=%{_prefix} --root=%{buildroot}
```

Comment:
Nit: while you're here (and in other specs), we can change this to `%py3_install` for better consistency.